qa/tasks/vstart_runner.py

```diff
@@ -530,8 +530,21 @@ def safe_kill(pid):
         else:
             raise
 
+def using_localhost(config_path="./ceph.conf"):
+    """
+    If the ceph cluster is using the localhost IP as mon host, will must disable ns unsharing
+    """
+    with open(config_path) as f:
+        for line in f:
+            asok_conf = re.search("^\s*mon\s+host\s*=\s*\[(.*:127.0.0.1:.*)+\]", line)
```

Comment:
Probably ought to escape the '.' characters in 127.0.0.1 as well, if you want to be extra careful. i.e.: `127\.0\.0\.1`

Reply:
Yeah, makes sense, I will fix it. Thanks.